conda-forge.yml

```diff
@@ -12,3 +12,4 @@ provider:
 test_on_native_only: true
 conda_build:
   pkg_format: '2'
+remote-ci-setup: ["conda-forge-ci-setup=3", "packaging"]  # temporary workaround
```

Comment:
Turns out the key was `remote_ci_setup` (underscores!), so no clue why this worked, or in other words, what else changed that made everything pass. But this change was NOT. We might prefer to revert this to avoid confusion. @conda-forge/conda-build

Reply:
Did it work? Not seeing it in CI scripts above. Am I missing something?